plugins/k8sauditeks/README.md

```diff
@@ -0,0 +1,79 @@
+# Kubernetes Audit Events Plugin for EKS
+
+## Introduction
+
+This plugin extends Falco to support [Kubernetes Audit Events](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-backends) from AWS EKS clusters as a new data source. 
+For more details about what Audit logs are, see the [README of k8saudit plugin](https://github.com/falcosecurity/plugins/blob/master/plugins/k8saudit/README.md).
+
+### Functionality
+
+This plugin supports consuming Kubernetes Audit Events stored in Cloudwatch Logs for the EKS Clusters, see [AWS official documentation](https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html) for details.
+
+## Capabilities
+
+The `k8sauditeks` uses the field extraction methods of the [`k8saudit`](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit) plugin as the format for the Audit Logs is same.
+
+### Event Source
+
+The event source for Kubernetes Audit Events from EKS is `k8s_audit`, it allows to use same rule than `k8saudit` plugin.
+
+### Supported Fields
+
+See the list [here](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit#supported-fields).
+
+## Usage
+
+### Configuration
+
+Here's an example of configuration of `falco.yaml`:
+
+```yaml
+plugins:
+  - name: k8sauditeks
+    library_path: libk8sauditeks.so
+    init_config:
+      region: "us-east-1"
```

Comment:
I see `region`, `profile`, and `cluster_name` as good candidates for open parameters instead than init configuration. Do you need these elsewhere out of the generation of events? If not, these are good open parameters for the event source.

Since that is usually a string, you should propose some URI-like encoding form.

Reply:
I'll switch them to `open_parameters`. 
What do you mean by "URI-like encoding form"?